Header header
float32 min_range
float32 max_range
float32 range
